src/api/todos.ts

```diff
@@ -6,3 +6,6 @@ export const getTodos = (userId: number) => {
 };
 
 // Add more methods here
+export const postTodo = (userId: number, todo: Omit<Todo, 'id'>) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, todo);
```

Comment:
You don't need a query param in your `post` it's defined in the API (check readme) 
```suggestion
  return client.post<Todo>(`/todos`, todo);
```